packages/react/src/components/Table/TableViewDropdown/TableViewDropdownItem.jsx

```diff
@@ -37,7 +37,7 @@ const TableViewDropdownItem = ({
   return (
     <div
       data-testid={testID}
-      title={`${text}${showEdited ? editedPostfix : ''}`}
+      title={tooltip || `${text}${showEdited ? editedPostfix : ''}`}
```

Comment:
We may want to add the same "edited" state like we do with the text to show when the view has been changed

Reply:
I updated custom tooltip so that it will also display edited state. It works locally, just having issue with deploy preview.

Reply:
Also, I added demo video to PR comment.